Helper/Cart.php

```diff
@@ -1725,7 +1725,9 @@ public function collectDiscounts(
             $discounts[] = [
                 'description' => trim(__('Discount ') . $address->getDiscountDescription()),
                 'amount'      => $roundedAmount,
-                'reference'   => $address->getCouponCode()
+                'reference'   => $address->getCouponCode(),
+                'discount_category' => 'coupon',
```

Comment:
Could you please add constants for discount_category, magento discount types and bolt discount types?

Reply:
Done